R/removeDirectories.r

```diff
@@ -17,49 +17,47 @@
 #' default)
 #' @return A logical vector the same length as the dirNames, indicating whether
 #' or not the corresponding directories were successfully removed
-#' @author Francisco Gochez
-#' @keywords IO
+#' @author Mike K Smith \email{mstoolkit@@googlemail.com}

```

Comment:
@author vs @contributor

Reply:
I already changed the author's name back.